screen_brightness_control/types.py

```diff
@@ -46,6 +46,7 @@
 - edid (str)
 - serial (str)
 - name (str)
+- uid (int)
```

Comment:
See earlier comment about UID uniqueness. If that list gets changed then this docstring should reflect the same order of operations